demo/text_to_text.py

```diff
@@ -22,4 +22,4 @@ def app():
 
 
 def upper_case_stream(s: str):
```

Comment:
Now that I think of it. I think the naming of the function is slightly misleading since it could be interpreted as upper casing all characters. Maybe we should change it to `capitalize_stream` here and the readme. Or we could change s.capitalize to s.upper here and in the readme.